README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
I think we are on the same page, there are four scenarios:
**(Full simulation mode)**
1. Clients runs Rviz, industrial_robot_simulator, generate_point_cloud 
2. Server runs all other processes
**(Real robot and simulate senor)**
1. Clients runs Rviz, robot_driver node, generate_point_cloud 
2. Server runs all other processes.
**(Simulated robot and Real senor)**
1. Clients runs Rviz, industrial_robot_simulator, Kinect sensor driver
2. Server runs all other processes.
**(Real robot and Real senor)**
1. Clients runs Rviz, robot_driver node, Kinect sensor driver
2. Server runs all other processes.

Now we have implemented scenario one.
(There is one issue that I ran industrial_robot_simulator on server side, I need to move this nodes to client side).
Before I was trying to say that:
like scenario one, do we need to separate client to work at different machine, like one machine only runs rviz, another machine runs industrial_robot_simulator, generate_point_cloud to act as simulated robot?  In the real world, if using real robot, if it's possible that the client machine running rviz is far away from the real robot?

Thank you for clarifying this. It is much clear now.